bberg_pil_cli/Cargo.toml

```diff
@@ -6,7 +6,7 @@ authors = ["Maddiaa"]
 edition = "2021"
 
 [[bin]]
-name = "bberg_pil"
+name = "bb_pil"
```

Comment:
I am fine with the renaming but please announce this change in the Slack channel. I have an alias currently pointing to bberg_pil and I would continue to use an old version until it breaks. I do not think that cargo build would clean the old binary.